Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CacheHelper.cs

```diff
@@ -85,7 +85,7 @@ public int GetCounterValue(string name, AzureWebApEnvironmentVariables environme
                     return 0;
                 }
 
-                CacheHelper.Instance.SaveToCache(name, uncachedJson, DateTimeOffset.Now.AddSeconds(5.0));
+                CacheHelper.Instance.SaveToCache(name, uncachedJson, DateTimeOffset.Now.AddMilliseconds(500));
```

Comment:
Quick pulse is actually requesting the values every second, so we have reduced the time to 500 milli seconds. Thought that 1 second might be too tight, as there could be a possibility of two requests happening consecutively with the timer timeouts.